ports/mimxrt/mpconfigport.h

```diff
@@ -97,7 +97,7 @@ uint32_t trng_random_u32(void);
 #define MICROPY_PY_MATH_ISCLOSE             (1)
 #define MICROPY_PY_CMATH                    (1)
 #define MICROPY_PY_IO_IOBASE                (1)
-#define MICROPY_PY_IO_FILEIO                (1)
+#define MICROPY_PY_IO_FILEIO                (MICROPY_VFS_FAT || MICROPY_VFS_LFS2)
```

Comment:
I think it's simpler to just leave this always enabled (otherwise, `MICROPY_VFS_LFS1` also needs to be included in the `||`)

Reply:
fixed.